Lib/zipfile.py

```diff
@@ -3,6 +3,7 @@
 
 XXX references to utf-8 need further investigation.
 """
+import collections
```

Comment:
For what it's worth, it is fairly expensive to import the `collections` module.

Reply:
It's really only [here](https://github.com/python/cpython/blob/3ed3f2495d6386f30cd3101059c51b0c997ae734/Lib/zipfile.py#L608) for aesthetic purposes. I could move it into the class definition to avoid importing it unnecessarily, or replace it with a less-pretty dict literal.